crates/store/re_entity_db/src/rrd_manifest_index.rs

```diff
@@ -18,11 +18,11 @@ pub struct ChunkInfo {
 ///
 /// This is currently used to show a progress bar.
 ///
-/// This is constructed from one ore more [`ChunkIndexMessage`], which is what
+/// This is constructed from one ore more [`RrdManifestMessage`], which is what
 /// the server sends to the client/viewer.
 /// TODO(RR-2999): use this for larger-than-RAM.
 #[derive(Default, Debug, Clone)]
-pub struct ChunkIndex {
+pub struct RrdManifest {
```

Comment:
I'm not a fan of re-using the exact same name as the actual `RrdManifest` object, this is confusing.